misc/demo/index.html

```diff
@@ -205,7 +206,7 @@
                         For the services (you will recognize them with the <code>$</code> prefix), we list all the possible parameters you can pass to them and their default values if any.
                     </p>
                     <p>
-                        Also, some components have default values that you can override globally within a <code>.config</code> function for example.
+                        * Some directives has a config service that follows the next pattern: <code>uibDirectiveConfig</code>. The settings on that service uses camel case. You can configure those services in a <code>.config</code> function for example.
```

Comment:
`Some directives have a config service...`
`The service's settings use camel case.`
`The services can be configured in...`